lighthouse/blueprints/reports.py

```diff
@@ -2,12 +2,14 @@
 from http import HTTPStatus
 from typing import Any, Dict, Tuple
 
-from flask import Blueprint
+from flask import Blueprint, request
 from flask_cors import CORS  # type: ignore
 
-from lighthouse.helpers.reports import get_reports_details
+from lighthouse.helpers.reports import get_reports_details, delete_reports
 from lighthouse.jobs.reports import create_report
 
+import json
```

Comment:
Is this being used

Reply:
Not anymore. I removed it's use as of this afternoon. I will remove it.